hoomd/jit/patch.py

```diff
@@ -195,6 +198,39 @@ def compile_user(self, code, clang_exec, fn=None):
 
         return llvm_ir
 
+    def set_alpha(self, alpha, index=None):
```

Comment:
Could we use an `alpha` property in place of `get_alpha`/`set_alpha`?:
```
>>> patch.alpha = [1,2,3]
>>> print(patch.alpha)
[1,2,3]
```

Reply:
For sure. Do you want to remove the set/get methods completely?